source/_posts/2025-07-25-nfc-floppy-disk-game-launcher.markdown

```diff
@@ -0,0 +1,249 @@
+comments: true
+categories: [Arduino, NFC, Retro Gaming, Hardware Hacking]
+---
+
+Remember the satisfying *click* of inserting a floppy disk and waiting for your favorite game to load? What if you could recreate that nostalgic experience while launching modern game collections? That's exactly what I built with my NFC Floppy Disk Game Launcher - a system that transforms vintage floppy disks into smart game cartridges using NFC technology.
+
+<!-- more -->
+
+## The Concept
+
+The idea is beautifully simple: take an old floppy disk drive, gut the internals, and replace them with an Arduino and NFC reader. Attach NFC stickers inside floppy disks, and suddenly you have physical "game cartridges" that can launch any game or application when inserted.
+
+{% video http://localhost:4000/images/posts/nfc-floppy/demo.mp4 400 600 %}
+
+This project bridges the gap between physical and digital gaming, creating a tactile experience that modern gaming has largely lost. Each floppy disk becomes a unique key to unlock specific games, scripts, or applications.
+
+## Hardware Components
+
+The beauty of this project lies in its simplicity. You'll need:
+
+- **Arduino Uno** (or compatible microcontroller)
+- **MFRC522 NFC/RFID module** (13.56MHz)
+- **Old floppy disk drive** (for the housing)
+- **NFC stickers/tags** (ISO 14443A compatible)
+- **Floppy disks** (as many as you want games)
+- **Connecting wires** and basic soldering supplies
+
+{% img center /images/posts/nfc-floppy/mfrc522.png 600 400 'Hardware Components' %}
+
+<!-- more -->
+
+### The MFRC522 Module
+
+The heart of the system is the MFRC522 NFC module. This little board operates at 13.56MHz and supports ISO/IEC 14443 Type A cards, MIFARE, and NTAG formats. With a typical reading range of 1-3cm, it's perfect for detecting tags inside floppy disks.
+
+**Pin Configuration:**
+- VCC → 3.3V (important: not 5V!)
+- GND → Ground  
+- RST → Digital Pin 9
+- SS → Digital Pin 10
+- MOSI → Pin 11 (SPI)
+- MISO → Pin 12 (SPI)
+- SCK → Pin 13 (SPI)
+
+## The Build Process
+
+### Step 1: Prepare the Floppy Drive
+
+First, I carefully disassembled an old 3.5" floppy drive, removing all the mechanical components while preserving the outer shell. The goal is to maintain the authentic look and feel of inserting a floppy disk.
+
+
+### Step 2: Mount the NFC Reader
+
+The MFRC522 module needs to be positioned where it can reliably read NFC tags inside inserted floppy disks. After some experimentation, I noticed the metalic shell for the floppy disk reader was interfering with the readings, so I had to fully remove all the mechanical parts. Because of that, I had to replicat ethe insertion mechanism, I did this with Lego !
+
+{% video http://localhost:4000/images/posts/nfc-floppy/lego-reader.mp4 400 600 %}
+
+### Step 3: Arduino Integration
+
+Then I connected the NFC reader to the Arduino, and the Arduino is connected it via USB to the computer for both power and serial communication.
```

Comment:
Grammar typo, fix:

Then I connected the NFC reader to the Arduino, which is itself connected via USB to the computer for both power and serial communication.